src/views/open-api-docs-preview-v2/utils/get-props-from-preview-data.ts

```diff
@@ -28,10 +34,44 @@ export default async function getPropsFromPreviewData(
 	if (!previewData) {
 		return null
 	}
-	// Use the incoming preview data to generate static props for the view
-	return await getStaticProps({
```

Comment:
I've updated `get-props-from-preview-data` so that we can show previewed specs more or less exactly how the production pages will appear. Specifically, I've added a schema transform here to replace `HashiCorp Cloud Platform` with `HCP` in the title (nearly all specs need this), as well as some generic resource links.

I've also hooked up the `Group operations by path` input from the form into the newly introduced `getOperationGroupKey` grouping control function.